src/Tribe/Editor/Compatibility.php

```diff
@@ -17,6 +30,164 @@ class Tribe__Events__Editor__Compatibility {
 	 */
 	public function hook() {
 		add_action( 'admin_init', array( $this, 'deactivate_gutenberg_extension_plugin' ) );
+		add_action( 'add_meta_boxes', array( $this, 'maybe_add_purge_blocks_metabox' ) );
+		tribe_notice(
+			'events-editor-compatibility-purge-blocks',
+			array( $this, 'purge_blocks_notice' ),
+			array(
+				'type'    => 'success',
+				'dismiss' => 1,
+				'wrap'    => 'p',
+			),
+			array( $this, 'should_display_purge_blocks_notice' )
+		);
+	}
+
+	/**
+	 * What is the notice when we purge blocks
+	 *
+	 * @since  TBD
+	 *
+	 * @return string
+	 */
+	public function purge_blocks_notice() {
+		return esc_html__( 'Clean up complete. This Event is now optimized for the classic editor.', 'the-events-calendar' );
+	}
+
+	/**
+	 * Check if we need to display the warning that we updated the Blocks to classic
+	 *
+	 * @since  TBD
+	 *
+	 * @return bool
+	 */
+	public function should_display_purge_blocks_notice() {
+		$action = tribe_get_request_var( 'tribe-action', false );
+
+		// Bail if incorrect action
+		if ( 'purge-event-blocks' !== $action ) {
+			return false;
+		}
+		$post = absint( tribe_get_request_var( 'post' ) );
```

Comment:
I think renaming this to provide a more meaningful context in this case `$post_id` as represents an ID other than a `WP_Post` object.